app/src/main/res/values/strings.xml

```diff
@@ -588,26 +589,27 @@
     <string name="share_url_more">Share URL…</string>
     <string name="copy_url">Copy URL to clipboard</string>
     <string name="media_attachment_empty">Missing media attachments.</string>
-    <string name="media_attachment_type_error">The type of media attachment is \"%1$s\". ST can\'t handle it, but you can try open in browser.</string>
+    <string name="media_attachment_type_error">Could not open the \"%1$s\" media attachment. Try to open it in a web browser.</string>

```

Comment:
Since there is an option of "give up to open" to the user, it is not made into an imperative form